src/Razor/test/Microsoft.AspNetCore.Razor.Test.ComponentShim/Microsoft.AspNetCore.Components/RenderTree/RenderTreeBuilder.cs

```diff
@@ -84,6 +84,10 @@ public void AddAttribute(int sequence, string name, object value)
         {
         }
 
+        public void AddMultipleAttributes<T>(int sequence, IEnumerable<KeyValuePair<string, T>> attributes)
```

Comment:
Interesting choice in names. I know this is just the shim but would have figured it'd be `AddAttributes`

Reply:
I didn't go that way because it doesn't seem distinct enough from `AddAttribute(...)`. This is a power tool and used rarely, `AddAttribute` is all over code that uses the builder.